### PR TITLE
Only Owner module

### PR DESCRIPTION
**Pull request by alyn509**
all endpoints within this module have `#[only_owner]`

### CODE REVIEW

File: contracts/feature-tests/basic-features/mandos/only_owner.scen.json [hunk 1]
```diff
@@ -61,6 +61,86 @@
                 "refund": "*"
             }
         },
+        {
```
Comment (andrei-marinica) [inline contracts/feature-tests/basic-features/mandos/only_owner.scen.json:64]: Can we also move this test to use-module?
I see it was removed from the mandos-rs suite

**Review by dorin-iancu — APPROVED** [review]

File: contracts/feature-tests/use-module/src/only_owner_mod.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+elrond_wasm::imports!();
+
+/// This module is in the crate, but it is not included.
+/// Its endpoints should not appear in the contract binary.
+#[elrond_wasm::only_owner_module]
```
Comment (andrei-marinica) [inline contracts/feature-tests/use-module/src/only_owner_mod.rs:5]: I would prefer to continue having
```
#[elrond_wasm::module]
#[only_owner]
```
Can we pull this off?

**Review by andrei-marinica — CHANGES_REQUESTED** [review]

File: contracts/feature-tests/basic-features/mandos/only_owner.scen.json (hunk 1 above)
Comment (alyn509) [reply contracts/feature-tests/basic-features/mandos/only_owner.scen.json]: wrote the test initially in basic-features. 
now I placed it in its right place

File: elrond-wasm-derive/src/model/argument.rs [hunk 3]
```diff
@@ -54,3 +54,14 @@ impl MethodArgument {
         matches!(self.metadata.payment, ArgPaymentMetadata::NotPayment)
     }
 }
+
+#[derive(Clone, Debug)]
+pub struct TraitProperties {
```
Comment (andrei-marinica) [inline elrond-wasm-derive/src/model/argument.rs:59]: Clippy says this can be derived

File: elrond-wasm-derive/src/parse/method_parse.rs [hunk 4]
```diff
@@ -30,7 +30,7 @@ pub fn process_method(m: &syn::TraitItemMethod) -> Method {
     let mut first_pass_data = MethodAttributesPass1 {
         method_name: m.sig.ident.to_string(),
         payable: MethodPayableMetadata::NotPayable,
-        only_owner: false,
+        only_owner: trait_attributes.only_owner.clone(),
```
Comment (andrei-marinica) [inline elrond-wasm-derive/src/parse/method_parse.rs]: Clippy: using `clone` on type `bool` which implements the `Copy` trait

**Review by andrei-marinica — APPROVED** [review]

**Pull request merged**